packages/api/cms-api/src/dam/files/files.utils.ts

```diff
@@ -1,12 +1,14 @@
 import { XMLParser } from "fast-xml-parser";
 import { unlink } from "fs/promises";
+import * as mimedb from "mime-db";
 import { sep } from "path";
 import slugify from "slugify";
 
 import { FileUploadInput } from "./dto/file-upload.input";
 
-export function slugifyFilename(filename: string, extension: string): string {
-    return `${slugify(filename, { locale: "de", lower: true, strict: true })}${extension}`;
+export function slugifyFilename(filename: string, extension?: string): string {
```

Comment:
Shouldn't extension be required here?

Reply:
Yes, you are right. I copied it back from admin where it makes sense that it's optional. But in the API it should be required: e32c7f17014b0c24275b518320c48fb84e2e2317